docs/kusion/2-getting-started/1-install-kusion.md

```diff
@@ -18,18 +13,27 @@ The recommended method for installing on MacOS and Linux is to use the brew pack
 **Install Kusion**
 
 ```bash
+# tap formula repository Kusionstack/tap
+brew tap KusionStack/tap
+
+# install Kusion 
 brew install KusionStack/tap/kusion
 ```
 
 **Update Kusion**
 
 ```bash
+# update formulae from remote
```

Comment:
> update formulae from remote

Here seems to be a typo, `formulae` ---> `formular`

Reply:
it's homebrew formulae, ref: https://formulae.brew.sh/